app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -102,6 +88,18 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
         inputLayoutPassword.isHintAnimationEnabled = false
 
         inputUsername.readOnly = true
+
+        if (showUsernamePlaceholder) {
+            view!!.btnUsernameCopy.setColorFilter(resources.getColor(R.color.white_60_percent))
+            view!!.isClickable = false
+            view!!.isFocusable = false
+            inputUsername.setText(getString(R.string.empty_space), TextView.BufferType.NORMAL)
+        } else {
+            inputUsername.isClickable = true
+            inputUsername.isFocusable = true
+            inputUsername.setText(item.username, TextView.BufferType.NORMAL)
+        }
```

Comment:
I'm still confused about this. I know that you're only copying code from where it was before to here. However, I don't understand the rationale for the code in the first place. 

```kotlin
btnUsernameCopy.setColorFilter(resources.getColor(R.color.white_60_percent))
view!!.isClickable = false
view!!.isFocusable = false
inputUsername.setText(R.string.empty_space, TextView.BufferType.NORMAL)
```

Intuitively, each side of this if/else should look very similar, if only to reverse the changes that successive calls make.

Shouldn't the `inputUsername` be non-clickable, and non-focusable, rather than the whole `view!!`?

Reply:
Good catch. This was leftover copy/paste from when this was a separate method. should be `inputUsername`